descriptors/ModuleDescriptor-template.json

```diff
@@ -707,6 +738,43 @@
         "eresource.item.delete"
       ]
     },
+    {
+      "permissionName" : "orders_storage_fund_distribution.collection.get",
```

Comment:
Please try to follow [Permissions Naming conventions](https://dev.folio.org/guidelines/naming-conventions/#permissions)
> The naming scheme is a faceted dot-separated string, with the delimited terms as dash-separated words. The first portion is the exact name of the responsible module (back-end modules drop the mod- prefix).

Reply:
Fixed according to naming conventions